drop-storage/src/lib.rs

```diff
@@ -1340,6 +1340,46 @@ impl Storage {
         }
     }
 
+    pub async fn fetch_temp_locations_for_file(
```

Comment:
It is a bit misleading, as this function fetches the base directories for downloads and has no knowledge of temporary files, it could be named `fetch_base_directories_for_file()` for example.

Question - do we expect multiple values? I assume it's just a failsafe just in case something went bad to remove the temp files from the previous downloads?

Reply:
Thanks :+1: . I will change the name. I'm not sure, but shouldn't be possible. However, it's safer to fetch all of them if present in the table

Reply:
Function name changed

Reply:
I agree that it's safer, lets leave it as it is :)